.gitignore

```diff
@@ -5,7 +5,6 @@
 assets/vendor/clipboard/src
 assets/vendor/clipboard/test
 assets/vendor/selectivizr/tests
-assets/vendor/qtip2/basic
```

Comment:
Seems like we could ignore hint.css/src?

Reply:
Yes @mlinksva , we can. Sorry, It skipped my mind before :sweat_smile: 
Updated and removed the ignored source files from the branch as well.